content/develop/concepts/configuration/theming-colors-and-borders.md

```diff
@@ -0,0 +1,126 @@
+
+The following configuration options can be set separately for the sidebar by using the `[theme.sidebar]` table instead of the `[theme]` table in `config.toml`:
+
+- `theme.primaryColor`
+- `theme.backgroundColor`
+- `theme.secondaryBackgroundColor`
+- `theme.textColor`
+- `theme.linkColor`
+- `theme.codeBackgroundColor`
+- `theme.baseRadius`
+- `theme.borderColor`
+- `theme.showWidgetBorder`
+
+For brevity, on the rest of this page, theming configuration options will not include the `theme.` or `theme.sidebar.` prefix.
+
+### `primaryColor`
+
+`primaryColor` defines the accent color most often used throughout your Streamlit
+app. The following features and effects use your primary color:
+
+- Button hover effects.
+- Elements in focus.
+- Selected elements.
+
+<Tip>
+
+When your primary color is used as a background, Streamlit changes the text color to white. For example, this happens for `type="primary"` buttons and for selected items in `st.multiselect`.
+
+For legibility, always choose a primary color that is dark enough to contrast well with white text.
+
+</Tip>
+
+### `backgroundColor`, `secondaryBackgroundColor`, and `codeBackgroundColor`
+
+`backgroundColor` defines the background color of your app. `secondaryBackgroundColor` defines the background color where contrast is needed.
+
+`secondaryBackgroundColor` is used in the following places:
+
+- The background of input or selection regions for widgets.
+- Headers within elements like `st.dataframe` and `st.help`.
+- Code blocks and inline code (if `codeBackgroundColor` is not set).
+
+`codeBackgroundColor` sets the background for code blocks and line code. If `codeBackgroundColor` is not set, Streamlit uses `secondaryBackgroundColor` instead.
+
+<Note>
+
+If you do not define background colors for the sidebar, Streamlit will will swap `backgroundColor` and `secondaryBackgroundColor` in the sidebar:
+
+- If `theme.sidebar.backgroundColor` is not defined, Streamlit uses `theme.secondaryBackgroundColor`.
+- If `theme.sidebar.secondaryBackgroundColor` is not defined, Streamlit uses `theme.backgroundColor`.
+
+</Note>
+
+### `textColor` and `linkColor`
+
+You can configure the color of body text and links.
+
+- `textColor` sets the default text color for all text in the app except language-highlighting in code blocks, inline code, and links.
+- `linkColor` sets the default font color for all Markdown links in the app.
+
+`st.page_link` and `st.link_button` use `textColor`, not `linkColor`. Also, as noted previously, Streamlit changes the text color to white when text is displayed on a background of your primary color.
+
+### `baseRadius`
+
+`borderRadius` defines the border radius on a variety of elements. Some of these elements do not have a visible border but are still affected by this configuration options because they do have a background color. The following elements are impacted by `baseRadius`:
```

Comment:
```defines the border radius on a variety of elements. Some of these elements do not have a visible border but are still affected by this configuration options because they do have a background color.```

That sounds a bit complex... I wouldn't even talk about the border but just say straight up that this defines the radius of elements.